server/django_server/settings.py

```diff
@@ -95,23 +109,34 @@
     }
 }
 
+# password hashing
+PASSWORD_HASHERS = (
+    'django.contrib.auth.hashers.PBKDF2PasswordHasher',
+    'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+    'django.contrib.auth.hashers.BCryptSHA256PasswordHasher',
+    'django.contrib.auth.hashers.BCryptPasswordHasher',
+    'django.contrib.auth.hashers.SHA1PasswordHasher',
+    'django.contrib.auth.hashers.MD5PasswordHasher',
+    'django.contrib.auth.hashers.CryptPasswordHasher',
+)
+
```

Comment:
Do we need all of these?

Sorry if this is a noob question, but I have the impression that we would only use like 1 or 2 hashers

Reply:
Yes i just forgot to remove them.